Frontend/ui-customer/src/Flow.purs

```diff
@@ -2401,10 +2423,23 @@ homeScreenFlow = do
         findEstimates state
     GOTO_CONFIRMING_LOCATION_STAGE finalState -> do
       liftFlowBT $ runEffectFn1 locateOnMap locateOnMapConfig { lat = finalState.props.sourceLat, lon = finalState.props.sourceLong, geoJson = finalState.data.polygonCoordinates, points = finalState.data.nearByPickUpPoints, labelId = getNewIDWithTag "LocateOnMapPin", zoomLevel = zoomLevel, locationName = fromMaybe "" finalState.props.locateOnMapProps.sourceLocationName, specialZoneMarkerConfig { labelImage = zoneLabelIcon finalState.props.confirmLocationCategory, labelMaxWidth = locateOnMapLabelMaxWidth} }
-      modifyScreenState $ HomeScreenStateType (\homeScreen -> homeScreen { props { currentStage = ConfirmingLocation, rideRequestFlow = true, locateOnMapLocation { sourceLat = finalState.props.sourceLat, sourceLng = finalState.props.sourceLong, source = finalState.data.source, sourceAddress = finalState.data.sourceAddress }, locateOnMapProps { cameraAnimatedToSource = false } } })
+      modifyScreenState $ HomeScreenStateType (\homeScreen -> homeScreen { props { currentStage = ConfirmingLocation, rideRequestFlow = true, locateOnMapLocation { sourceLat = finalState.props.sourceLat, sourceLng = finalState.props.sourceLong, source = finalState.data.source, sourceAddress = finalState.data.sourceAddress }, locateOnMapProps { cameraAnimatedToSource = false } , isSource = Just true,locateOnMap = false}})
       void $ pure $ updateLocalStage ConfirmingLocation
       void $ lift $ lift $ toggleLoader false
       homeScreenFlow
+    GO_TO_CONFIRMING_DROP_LOCATION_STAGE finalState -> do
```

Comment:
`GO_TO_CONFIRMING_DROP_LOCATION_STAGE` and  `GOTO_CONFIRMING_LOCATION_STAGE` both have similar blocks of code can we combine these two.

Reply:
while existing sent some tag whether its source or destination in the output. use that to unify this.